pages.tr/common/fzf.md

```diff
@@ -0,0 +1,29 @@
+# fzf
+
+> Komut satırı belirsiz bulucu.
+> Sk'ya benzer.
+> Daha fazla bilgi: <https://github.com/junegunn/fzf>.
```

Comment:
We have a template for more information, see https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py#L41 .
```suggestion
> Daha fazla bilgi için: <https://github.com/junegunn/fzf>.
```

Reply:
That's a minor one, it's ok will use it next time.